summary_workflows/quantification/quantification_dockers/q_consolidation/aggregation.py

```diff
@@ -144,10 +144,17 @@ def main():
 
         logging.debug(f"aggregation after update: {new_aggregation}")
 
-        # 2.e) Write aggregation file per challenge to local results dir
-        aggregation_file = os.path.join(challenge_dir, challenge_id + ".json")
-        with open(aggregation_file, mode='w', encoding="utf-8") as f:
-            json.dump(new_aggregation, f, sort_keys=True, indent=4, separators=(',', ': '))
+        # 2.e) Write each aggregation object to a separe file per challenge to local results dir
+        for count, aggr_object in enumerate(new_aggregation):
```

Comment:
@AsierGonzalez I'm afraid with this approach the `consolidated_result.json` will also only contain the first aggregation for each challenge. However, if I understood correctly, the web visualization of OEB *can* handle multiple aggregations per challenge and thus create multiple plots per challenge. Actually the whole exercise of having the branch `swf_multiple_plots` was to be able to submit a `consolidated_result.json` with *multiple* aggregation objects per challenge to the OEB DB.    
As I see it we have several options:   
- discard the VRE visualization and just use the web viz, as it was without the present PR
or
- adapt the script `merge_data_model_files.py` to loop over all those numbered aggregation files that you create, so that they are united again in the `consolidation_results.json`
or
- write the aggregation file with multiple objects *in addition* to the numbered ones, and only load that "multiple objects file" in the `merge_data_model_files.py`

Reply:
I think I fixed it.